suse_migration_services/units/prepare.py

```diff
@@ -162,13 +172,14 @@ def main():
         system_mount.add_entry(
             zypp_plugins_services, '/usr/lib/zypp/plugins/services'
         )
-        if os.path.exists(cloud_register_metadata):
-            log.info('Bind mounting /var/lib/cloudregister')
-            Path.create('/var/lib/cloudregister')
+        if cloud_register_metadata and os.path.exists(cloud_register_metadata):
```

Comment:
this condition can be
```python
if os.path.exists(path)
```
the double check does not add safety or value,
if code reaches that part and `cloud_register_metadata` has not been populated, checking as it is:
```python
if path and os.path.exists(path)
```
produces an exception same as `if os.path.exists(path)` 
there needs to be a new test similar to `test_main_raises_on_get_regionsrv_client_file_location` but with a different value (for example `foo`) [here](https://github.com/SUSE/suse-migration-services/pull/251/files#diff-244e5e611b27f16d2db5163a4a3806e6352bf26d6ca8fbf2ca4a3bac2bd4ea2bR69) to capture this scenario

adding
```python
cloud_register_metadata = ""
with open(hosts_setup, 'r', encoding="utf-8") as fp:
      ....
```
is enough
```python
cloud_register_metadata = ""
os.path.exists(cloud_register_metadata)
False
```

Reply:
i will  `cloud_register_metadata = ""` in the code right after i defined `cache_cloudregister_path = '/var/cache/cloudregister'`